_releases/2023-04-12-1.8.0-released.md

```diff
@@ -0,0 +1,128 @@
+- OpenSSL: use Windows' system root certificate store ([#13187](https://github.com/crystal-lang/crystal/pull/13187), thanks @HertzDevil)
+- ⚠️ Add full stub for Windows signals ([#13131](https://github.com/crystal-lang/crystal/pull/13131), thanks @HertzDevil)
+
+## Signals
+
+By adding support to Windows signals ([#13131](https://github.com/crystal-lang/crystal/pull/13131)), it become necessary to improve the signals API to permit a patform-independent way to handle them.  Here is the list of changes in this regard:
+
+- Add [`Process.on_interrupt`](https://crystal-lang.org/api/1.8/Process.html#on_interrupt%28%26handler%3A-%3E%29%3ANil-class-method) to handle `SIGINT` (Unix) and `Ctrl+C` or `Ctrl+Break` (Windows) ([#13034](https://github.com/crystal-lang/crystal/pull/13034), thanks @HertzDevil).
+- Add `graceful` parameter to [`Process#terminate`](https://crystal-lang.org/api/1.8/Process.html#terminate%28%2A%2Cgraceful%3ABool%3Dtrue%29%3ANil-instance-method) ([#13070](https://github.com/crystal-lang/crystal/pull/13070), thanks @HertzDevil).
+- Add [`Process::ExitReason`](https://crystal-lang.org/api/1.8/Process/ExitReason.html) and [`Process::Status#exit_reason`](https://crystal-lang.org/api/1.8/Process/Status.html#exit_reason%3AExitReason-instance-method) ([#13052](https://github.com/crystal-lang/crystal/pull/13052), thanks @HertzDevil).
+
+## Misc
+
+### Language
+
+- Allow namespaced `Path`s as type names for `lib` ([#12903](https://github.com/crystal-lang/crystal/pull/12903), thanks @HertzDevil).
+
+### Compiler
+
+### stdlib
+
+- Fix: LibC definition for `sys/socket.h` (`*-linux-gnu` targets) ([#13242](https://github.com/crystal-lang/crystal/pull/13242), thanks @ysbaddaden)
+
+- Handle `Range` requests in `HTTP::StaticFileHandler` ([#12886](https://github.com/crystal-lang/crystal/pull/12886), thanks @jgaskins, @straight-shoota)
+
+- Fix JSON, YAML `use_*_discriminator` for recursive struct types ([#13238](https://github.com/crystal-lang/crystal/pull/13238), thanks @HertzDevil)
+
+- Format spec results with pretty inspect ([#11635](https://github.com/crystal-lang/crystal/pull/11635), thanks @JamesGood626)
+- Spec: Add `--color` option to spec runner ([#12932](https://github.com/crystal-lang/crystal/pull/12932), thanks @straight-shoota)
+
+#### Enum
+
+- ⚠️ Fix `Enum#includes?` to require all bits set ([#13229](https://github.com/crystal-lang/crystal/pull/13229), thanks @straight-shoota)
+- ⚠️ Deprecate `Enum.flags` ([#12900](https://github.com/crystal-lang/crystal/pull/12900), thanks @straight-shoota)
+- Add `Enum.[]` convenience constructor ([#12900](https://github.com/crystal-lang/crystal/pull/12900), thanks @straight-shoota)
+- Add `Enum#inspect` ([#13004](https://github.com/crystal-lang/crystal/pull/13004), thanks @straight-shoota)
+
+#### Range
+
+- ⚠️ Remove compile-time error for `Range#size`, `#each`, `#sample` ([#13278](https://github.com/crystal-lang/crystal/pull/13278), thanks @straight-shoota)
```

Comment:
Expanding a bit the description.
```suggestion
- ⚠️ Remove compile-time error for `Range#size`, `#each`, `#sample`. For instance, `(..4).each { }` will now raise an exception instead of failing at compile time ([#13278](https://github.com/crystal-lang/crystal/pull/13278), thanks @straight-shoota).
```